win32ss/user/user32/misc/exticon.c

```diff
@@ -422,7 +422,11 @@ static UINT ICO_ExtractIconExW(
 	sig = USER32_GetResourceTable(peimage, fsizel, &pData);
 
 /* NE exe/dll */
+#ifdef __REACTOS__
+    if (sig==IMAGE_OS2_SIGNATURE || sig==1) /* .ICO file */
```

Comment:
```suggestion
/* NE exe/dll: IMAGE_OS2_SIGNATURE
   .ICO file: 1
 */
#ifdef __REACTOS__
    if (sig==IMAGE_OS2_SIGNATURE || sig==1)
```
Something along those lines is a bit more clear in my opinion.